<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Visualization and Analysis of Spatial Heterogeneity in Spatially-Resolved Gene Expression • spatialGE</title>



      </header><div class="row">
  <div class="contents col-md-9">
<div class="section level1">
<div class="page-header"><h1 id="spatialge">spatialGE<a class="anchor" aria-label="anchor" href="#spatialge"></a>
</h1></div>
<p>An R package for the visualization and analysis of spatially-resolved transcriptomics data, such as those generated with 10X Visium. The <strong>spatialGE</strong> package features a data object (STlist: Spatial Transctiptomics List) to store data and results from multiple tissue sections, as well as associated analytical methods for:</p>
<ul>
<li>Visualization: <code>STplot</code>, <code>gene_interpolation</code>, <code>STplot_interpolation</code> to explore gene expression in spatial context.</li>
<li>Spatial autocorrelation: <code>SThet</code>, <code>compare_SThet</code> to assess the level of spatial uniformity in gene expression by calculating Moran’s I and/or Geary’s C and qualitatively explore correlations with sample-level metadata (i.e., tissue type, therapy, disease status).</li>
<li>Tissue domain/niche detection: <code>STclust</code> to perform spatially-informed hierarchical clustering for prediction of tissue domains in samples.</li>
<li>Gene set spatial enrichment: <code>STenrich</code> to detect gene sets with indications of spatial patterns (i.e., non-spatially uniform gene set expression).</li>
<li>Gene expression spatial gradients: <code>STgradient</code> to detect genes with evidence of variation in expression with respect to a tissue domain.</li>
<li>Spatially-informed differential expression: <code>STdiff</code> to test for differentially expressed genes using mixed models with spatial covariance structures to account of spatial dependency among spots/cells. It also supports non-spatial tests (Wilcoxon’s and T-test).</li>
</ul>
<p>The methods in the initial spatialGE release, technical details, and their utility are presented in this publication: <a href="https://doi.org/10.1093/bioinformatics/btac145" class="external-link uri">https://doi.org/10.1093/bioinformatics/btac145</a>. For details on the recently developed methods <code>STenrich</code>, <code>STgradient</code>, and <code>STdiff</code> please refer to the spatialGE documentation.</p>
<p align="center">

  </div>



  </body>
</html>
